musicbrainz-server/templates/default/sv-musicbrainz-ws-run.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+echo starting
+exec 2>&1
+set -e
+cd mb_server
+umask 022
+exec \
+    env MUSICBRAINZ_USE_PROXY=1 starman -Ilib \
+        --preload-app \
+        -E deployment \
+        --listen musicbrainz-ws.socket \
+        --workers <%= @variables[:nproc] || 5 %> \
```

Comment:
Yeah, from what I understand the `variables :nproc => ` thing in the service description rebinds it here.